src/utilities/balanceChanges/balanceChanges.ts

```diff
@@ -3,30 +3,27 @@ import { ConfigService } from '@kiltprotocol/sdk-js';
 
 import { ErrorFirstCallback } from '../../channels/base/types';
 
-import { transformBalances } from '../transformBalances/transformBalances';
+import {
+  BalancesV1,
+  BalancesV2,
+  transformBalances,
+} from '../transformBalances/transformBalances';
 import { exceptionToError } from '../exceptionToError/exceptionToError';
 
-export interface Balances {
-  free: BN;
-  miscFrozen: BN;
-  feeFrozen: BN;
-  reserved: BN;
-}
-
 export interface BalanceChange {
   address: string;
   balances: {
```

Comment:
We could define this type via `TransformedBalances` instead of defining this structure once again